OpenKh.Tools.Kh2TextEditor/ViewModels/TextEditorViewModel.cs

```diff
@@ -127,6 +139,66 @@ public TextEditorViewModel()
             Drawing = new SpriteDrawingDirect3D();
             CurrentMessageEncoder = Encoders.InternationalSystem;
             _messages = new MessagesModel(this, this, new Msg.Entry[] { });
+
+            HandleReturn = new RelayCommand(x =>
+            {
+                Text += "{:newline}";                
+                SelectedItem = SelectedItem;
+            });
+
+            HandleRemoval = new RelayCommand(x =>
+            {
+                MessageEntries.RemoveAll(x => x.Id == SelectedItem.Id);
```

Comment:
Small note: `SelectedItem` can return `null`.

Reply:
Would be easy to fix. I did not realize this.